PostHog/UIViewController.swift

```diff
@@ -29,17 +29,9 @@
             // if a view is being dismissed, this will return nil
             if let root = viewIfLoaded?.window?.rootViewController {
                 return root
-            } else {
-                // preferred way to get active controller in ios 13+
-                for scene in UIApplication.shared.connectedScenes where scene.activationState == .foregroundActive {
-                    let windowScene = scene as? UIWindowScene
-                    let sceneDelegate = windowScene?.delegate as? UIWindowSceneDelegate
-                    if let target = sceneDelegate, let window = target.window {
```

Comment:
I'm not 100% sure but I believe the reason to check the target `delegate` is that the delegate has the `rootViewController` but on the other approach not but I don't remember for sure, better to test this before causing a regression, otherwise all good dropping the check.

Reply:
> // if a view is being dismissed, this will return nil

there's this comment, I believe its because of that, and in this case, maybe we should use `filterForegrounded: false`? its a fallback anyway.

Reply:
> I think we should consider removing the check on UIScene.activationState entirely and instead rely on UIApplication.shared.applicationState to determine whether to skip certain screen captures or not.

good point, I think we can do a follow up on that later so we don't drag this PR too much